code/game/machinery/vending/vendor_types/squad_prep/squad_prep.dm

```diff
@@ -71,6 +71,7 @@
 	listed_products = list(
 		list("STANDARD EQUIPMENT", -1, null, null, null),
 		list("Marine Combat Boots", round(scale * 15), /obj/item/clothing/shoes/marine, VENDOR_ITEM_REGULAR),
+		list("Jungle Pattern Combat Boots", round(scale * 1), /obj/item/clothing/shoes/marine/jungle, VENDOR_ITEM_REGULAR),
```

Comment:
These need to stay as buyable accessories, not standard issue

Reply:
you might want to take a look at miscelanous in the clothing vendor line 149 then...